test/storage/export_table_test.cpp

```diff
@@ -223,7 +223,7 @@ TEST_F(ExportTableTest, ExportDictionaryCompressedTableTest) {
   // Technically, the block above is not "in" the table, but since we don't sequential scan that does not matter
   storage::DataTable table(common::ManagedPointer<storage::BlockStore>(&block_store_), layout,
                            storage::layout_version_t(0));
-  storage::RawBlock *block = table.begin()->GetBlock();
+  storage::RawBlock *block = *table.GetBlocks().begin();
```

Comment:
Unintended change?

Reply:
No, in the empty table we have one block with no tuples. In the iterator now, when the `SlotIterator` is equal to `end()`, the `current_slot_` is the `InvalidTupleSlot`. So the old code will yield `nullptr` since it points to an invalid tuple slot. Getting the blocks instead returns all the blocks in the table even if some of them are currently empty. So we use that instead here.